core/src/test/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriterSuite.java

```diff
@@ -98,7 +98,7 @@ public void tearDown() {
   @Before
   @SuppressWarnings("unchecked")
   public void setUp() throws IOException {
-    MockitoAnnotations.initMocks(this);
+    MockitoAnnotations.openMocks(this);
```

Comment:
Shouldn't we do `openMocks(this).close()`? that's what the docs say.

Reply:
After tracing the code, I found that the `close()` method alway `do nothing`, but I'll change them all to `openMocks(this).close()` later :)

Reply:
a8ebb43 fix these